src/codeManager.ts

```diff
@@ -297,6 +297,11 @@ export class CodeManager implements vscode.Disposable {
     private getWorkspaceRoot(codeFileDir: string): string {
         return this._workspaceFolder ? this._workspaceFolder : codeFileDir;
     }
+    
+    private getWorkspaceName(workspaceRoot: string): string {
+        const regexMatch = workspaceRoot.match(/([^\/\\]+)[\/\\]?$/);
+        return regexMatch ? regexMatch[1] : this._codeFile;

```

Comment:
this._codeFile --> codeFileDir?

Reply:
In an ideal situation, the regular expression always succeeds in matching. Because it depends on `getWorkspaceRoot`... In a separate, how about returning `workspaceRoot` instead of `this._codeFile`?